pyproject.toml

```
@@ -454,14 +454,19 @@ explicit = true
 name = 'test.pypi'
 url = 'https://test.pypi.org/simple'
 
+[[tool.uv.index]]
+explicit = true
+name = 'gt4py_python_index'
+url = 'https://gridtools.github.io/python-pkg-index/'
+
+# -- versioningit --
 [tool.uv.sources]
 atlas4py = {index = "test.pypi"}
 dace = [
   {git = "https://github.com/GridTools/dace", branch = "romanc/stree-roundtrip", group = "dace-cartesian"},
-  {git = "https://github.com/GridTools/dace", tag = "__gt4py-next-integration_2025_12_11", group = "dace-next"}
+  {index = "gt4py_python_index"}
```

I think this is still needed, at least until dace-cartesian version is not moved to the gridtools package index
```suggestion
  {index = "gridtools", group = "dace-next"}
```